### PR TITLE
Create temporary sw plugin (no transpile)

### PR DESCRIPTION
**Pull request by angus-c**

### CODE REVIEW

File: src/browser.js [hunk 1]
```diff
@@ -5,18 +5,20 @@
...
+          .register('/sw.js')
+          .then(function(res) {
+            return console.log('*** sw registered:', res);
+          })
+          .catch(function(e) {
+            return console.log('*** sw registration failed:', e);
+          });
+      });
+    }
+    return null;
```
Comment (KevinGrandon) [inline src/browser.js]: Is this necessary?

Comment (lhorie) [reply src/browser.js]: shouldn't be @angus-c

**Comment by angus-c** [comment]
!merge

**Comment by angus-c** [comment]
@KevinGrandon I'm going to make it a pre-release (+ only pushed to a branch) and give to eats as agreed at last standup

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**